examples/creatives/base_simid_creative.js

```diff
@@ -172,4 +173,9 @@ class BaseSimidCreative {
   onVolumeChange(data) {
     this.videoState.volume = data.args.volume;
   }
+
+  onReceivePlayerLog(data) {
+    const logMessage = data.args.message;
+    console.log(logMessage);
```

Comment:
I wonder if maybe we should prefix this with something?
Like maybe `"Received message from player: " + logMessage`?

Reply:
good idea